src/wp-includes/class-wp-script-modules.php

```diff
@@ -148,6 +160,21 @@ public function enqueue( string $id, string $src = '', array $deps = array(), $v
 		}
 	}
 
+	/**
+	 * Marks the script module for inclusion in the import map.
+	 *
+	 * This method is intended for use outside of the script module dependency system.
+	 * It's recommended that script modules rely on the script module dependency system
+	 * to manage the import map.
+	 *
+	 * @since 6.8.0
+	 *
+	 * @param string $id The identifier of the script module.
+	 */
+	public function include_in_import_map( string $id ) {
```

Comment:
I wonder if this method name is too specific and tied to the implementation.

Originally I had used `expose()` and was also considering `make_available()`. I'm wondering about a future where this may add some option arguments for preload, etc.

Reply:
My first thought was `add_to_import_map()`